website/docs/r/data_factory_linked_service_sql_server.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_factory_linked_service_sql_server"
+sidebar_current: "docs-azurerm-resource-data-factory-linked-service-sql-server"
+description: |-
+  Manage an Azure Data Factory Linked Service SQL Server.
+---
+
+# azurerm_data_factory_linked_service_sql_server
+
+Manage an Azure Data Factory Linked Service SQL Server.
+
+## Example Usage
```

Comment:
since there's a connection string here we should also include the State file disclaimer:

```
~> **Note:** All arguments including the client secret will be stored in the raw state as plain-text. [Read more about sensitive data in state](/docs/state/sensitive-data.html).
```